server/etvas-events/product-canceled.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * This handler is called when the product.canceled event is emitted
+ * in ETVAS Platform. The app receives this event as a POST request,
+ * on the URL provided in Platform at product setup. The SDK
+ * automatically routes the event to this handler, which receives
+ * the payload as an object containing userId and purchaseId. Both
+ * of them are UUID v4, 36 character strings.
+ *
+ * The SDK will also verify the identity of the caller for you, so
+ * you know you're safe!
+ *
+ * Remember, the purchaseId can be used as context in headers when
+ * calling ETVAS Platform API directly or with the help of SDK.
+ */
+const etvas = require('@etvas/etvas-sdk')
+
+/**
+ * Product Canceled handler function
+ *
+ * @param {object} payload The event payload
+ * @returns True or object
+ */
+const handler = async ({ userId, purchaseId }) => {
+  // If you previously stored your internal ID in ETVAS
+  // platform, you can retrieve and co a cleanup on your side
+  const { internalId } = etvas.client.read(purchaseId)
```

Comment:
let's mention you can also clean up based on purchaseId

Reply:
It's on line 34

Reply:
that is something else .. that clears data on etvas .. 
usually we expect integrations to cleanup with 

`await myApi.removeSubscription(purchaseId)`